packages/esm-patient-list-management-app/src/lists-table/custom-pagination.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
This wrapper component around the Carbon [Pagination](https://react.carbondesignsystem.com/?path=/docs/components-pagination--overview) component is drawn from the Patient Chart and should probably be made more widely available elsewhere. It matches the related [designs](https://zpl.io/brGXRqX) better than the default Pagination component out of the box.

Reply:
Yep. We should probably ticket this to add an `OpenmrsPagination` component or something similar to the styleguide.

I think the rule here is something like: if the designs call for a component in more than one app that is not achievable without custom DOM and CSS, it should be a custom component.